src/index.css

```diff
@@ -1,22 +1,13 @@
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+:root {
+  font-family: var(--n-font-family);
+  color: var(--n-color-text);
+  background-color: var(--n-color-surface);
```

Comment:
For the sake of providing better defaults, should we also include:

```css
font-feature-settings: var(--n-font-features);
font-weight: var(--n-font-weight);
line-height: var(--n-line-height);
 ```

Reply:
Will do!

Reply:
Added 👍